migrations/20201015005801-update-user-roles.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'users',
+          'roleId',
+          { type: Sequelize.DataTypes.INTEGER },
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          `UPDATE users SET "roleId" = 1 WHERE "accessLevel" = 'COMMANDER' OR "accessLevel" = 'ADMIN'`,
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          `UPDATE users SET "roleId" = 2 WHERE "accessLevel" = 'SUPERVISOR'`,
+          { transaction: t }
+        ),
+        queryInterface.removeColumn('users', 'accessLevel', {
+          transaction: t,
+        }),
+      ]);
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'users',
+          'accessLevel',
+          {
+            type: Sequelize.DataTypes.STRING,
+          },
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          `UPDATE users SET "accessLevel" = 'COMMANDER' WHERE "roleId" = 1`,
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          `UPDATE users SET "accessLevel" = 'SUPERVISOR' WHERE "roleId" = 2`,
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
+          `UPDATE users SET "accessLevel" = 'DISPATCH' WHERE "roleId" = 3`,
```

Comment:
we don't have the `DISPATCH` access level before so this wouldn't work - we can swap it to `SUPERVISOR` instead (not ideal but i think this is the best option)
```suggestion
          `UPDATE users SET "accessLevel" = 'SUPERVISOR' WHERE "roleId" = 3`,
```

Reply:
well there shouldn't be any data with `roleId=3` so this line probably won't run, it's more just to be explicit. `roleId=2` is already `SUPERVISOR`, so don't think it's correct to map those

Reply:
Actually I ran into this issue when I was doing my maps ticket. I accidentally used your backend and tried to migrate down, but I had `DISPATCH` in my access levels.

Reply:
right - what exactly is the issue with mapping `DISPATCH` to `roleId = 3` in the down migration though? If I add a mapping for `roleId=3` => `DISPATCH` in the up migration everything should be fine?